### ISSUE TITLE
Web and Wasm target.

### ISSUE THREAD

**Issue by JADSN** [fruiframework/frui#11]
I like very of the approach of using Flutter concepts to mount the UI. 

This project will give support for webassembly support?

I also like the approach of [async-ui](https://github.com/wishawa/async_ui). 

As a suggestion, it would be good if you put the concepts together.

I make a issue here, because I couldn't join discord. **Sorry**.

Thanks.

**Comment by xanahopper** [comment]
Yes, multiple platform target is one of the goals. But it does not have a high priority.

(IMO, I'd like mobile Android and iOS, but it's too early to say that.

**Comment by toiglak** [comment]
Hey! Thank you for being interested in Frui!

> This project will give support for webassembly support?

Yes it will! As Xana mentioned, we're currently focusing on the most essential parts of the framework and it is not a priority. If you have some spare time, you can check out how [`druid`](https://github.com/linebender/druid) renders to WASM and try adapting it work with Frui (because Frui and Druid share the same back-end). Though it is totally fine if we do this later 😃 .

> I also like the approach of [async-ui](https://github.com/wishawa/async_ui).

`async-ui` is awesome and I think we can learn a lot from it! 

If you want to, you can share with us what you consider especially useful in it, so we can brainstorm ideas together! 

> I make a issue here, because I couldn't join discord.

My bad! I made a temporary invite instead of a permanent one. Should be fixed now. Come join us! 😄 

<a href="https://discord.gg/CCVygCAgXR"><img src="https://img.shields.io/badge/Join-flat.svg?label=Discord&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2" alt="Discord"/></a>

**Comment by JADSN** [comment]
If you want to, you can share with us what you consider especially useful in it, so we can brainstorm ideas together!

> It will be useful to create a plugin for [Tauri](https://tauri.app/), because in future this tech(based on roadmap) will target Mobile besides Desktop. I will like to use Frui for Web using the techs like [Yew](https://yew.rs/) and [Trunk](https://trunkrs.dev/).
> 
> At the moment, I'm trying to use this [article](https://dev.to/stevepryde/create-a-desktop-app-in-rust-using-tauri-and-yew-2bhe) to do something with `async-ui` and `Tauri`.
> 
> Also, Frui will help me make excellent UI's without the knownledge of CSS.
